test/e2e/specs/editor/various/is-typing.spec.js

```diff
@@ -1,46 +1,43 @@
 /**
  * WordPress dependencies
  */
-import {
-	clickBlockAppender,
-	createNewPost,
-	showBlockToolbar,
-} from '@wordpress/e2e-test-utils';
-
-describe( 'isTyping', () => {
-	beforeEach( async () => {
-		await createNewPost();
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'isTyping', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
 	} );
 
-	it( 'should hide the toolbar when typing', async () => {
+	test( 'should hide the toolbar when typing', async ( { page, editor } ) => {
 		const blockToolbarSelector = '.block-editor-block-toolbar';
```

Comment:
Rather than using a locator multiple times below, it'd be good to just use the locator once here:
```js
		const blockToolbar = page.locator( '.block-editor-block-toolbar' );
```

And then the code below would be rewritten to something like this:
```js
expect( blockToolbar ).toBeHidden();
```

It should be possible to use a role selector. Here are the accessibility properties:
![Screen Shot 2022-09-15 at 4 47 30 pm](https://user-images.githubusercontent.com/677833/190359165-848c8cf2-b421-4a1b-a91e-ad62c512cb9e.png)

So I think the role would be 'toolbar' and the name would be 'Block tools'